config/config/common-config.yaml

```diff
@@ -1103,3 +1111,40 @@ nodeLocalDns:
     requests:
       cpu: 25m
       memory: 40Mi
+
+externalDns:
```

Comment:
Optional: Do we want to expose the usual suspects `affinity` `nodeSelector` (might be redundant), `tolerations` and `resources`?

Reply:
I forgot to add resources. But tolerations, affinity/nodeSelector are probably also a good idea.